### PR TITLE
feat(cache): properly handle http caching headers

### PR DESCRIPTION
**Pull request by JacopoDaeli**

### CODE REVIEW

File: lib/routes/objects.js
Comment (decompil3d) [inline lib/routes/objects.js:100]:
```suggestion
            // cached for longer time since it's unlikely to change
```

**Review by decompil3d — APPROVED** [review]
Tiny comment spelling nit

File: lib/routes/objects.js
Comment (JacopoDaeli) [reply lib/routes/objects.js:100]: I know something was not right here ah ah. Thanks

**Comment by JacopoDaeli** [comment]
Waiting for @decompil3d review before merging this.

**Comment by JacopoDaeli** [comment]
Waiting for @Swaagie before merging this.

**Review by decompil3d — APPROVED** [review]

**Pull request merged**
